qiskit_nature/mappers/second_quantization/qubit_mapper.py

```diff
@@ -141,6 +141,9 @@ def mode_based_mapping(
                     raise QiskitNatureError(
                         f"FermionicOp label included '{char}'. Allowed characters: I, N, E, +, -"
                     )
+                if ret_op.size >= 100:
```

Comment:
How did you arrive at 100 being a good choice here? Did you perform experiments?

Reply:
I tested it with the microbenchmark https://github.com/Qiskit/qiskit-nature/pull/549#issuecomment-1039787299. But, I want actual cases hopefully as part of app-benchmarks.

Reply:
Cool. Perhaps we could include this information as a comment here.